tests/test_project.py

```diff
@@ -2116,6 +2125,20 @@ def test_main_run(self):
             assert job.doc.op2
             assert job.doc.op3
 
+    def test_main_run_abbreviated(self):
+        project = self.mock_project()
+        assert len(project)
+        for job in project:
+            assert not job.doc.get("op2", False)
+            assert not job.doc.get("op3", False)
+
+        # Use an abbreviated aggregate id
+        self.call_subcmd(f"run -o group_agg -j {get_aggregate_id(project)[:-5]}")
```

Comment:
I think adding a test for duplicates here might make sense.
```
self.call_subcmd(f"run -o some_operation -j {get_aggregate_id(project)} {get_aggregate_id(project)[:-5]}")
```
We should expect the aggregate operation to run just once.